content/it/_index.md

```diff
@@ -5,23 +5,41 @@ status: Completed
 
-Chiunque voglia contribuire può aprire una issue o una pull request su GitHub. Leggi la sezione [Come Contribuire](/it/contribute/) per maggiori informazioni e assicurati di seguire le indicazioni della [Guida di Stile](/it/style-guide/).
+Chiunque voglia contribuire può aprire una issue o una pull request su GitHub.
+Leggi la sezione [Come Contribuire](/it/contribute/) per maggiori informazioni e assicurati di seguire le indicazioni della [Guida di Stile](/it/style-guide/). Iscriviti al canale [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) dello slack del CNCF.
+C'è anche un canale [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) per coloro che vogliono aiutare nella traduzione del glossario nella loro madrelingua.
 
 ## Riconoscimenti
 
-Il progetto del Glossario Cloud Native è un'iniziativa del Marketing Committee della CNCF (nello specifico del Business Value Subcommittee) e include contributi da [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
-[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) e molti altri. Per la lista completa fai riferimento a [questa pagina Github](https://github.com/cncf/glossary/graphs/contributors).
-
-La localizzazione del progetto in italiano è a cura di [Giulia Di Pietro](https://www.linkedin.com/in/giulia-dp/), [Meryem Fourdaous](https://www.linkedin.com/in/meryem-fourdaous-022346102/), [Annalisa Gennaro](https://www.linkedin.com/in/annalisagennaro/) [Francesco Sbaraglia](https://www.linkedin.com/in/fsbaraglia/) e [Simone Stella](https://www.linkedin.com/in/simostella/).
+Il progetto del Glossario Cloud Native è un'iniziativa del Marketing Committee della CNCF (nello specifico del Business Value Subcommittee) e include contributi da 
+[Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), 
+[Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), 
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), 
+[Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), 
+[Katelin Ramer](https://www.linkedin.com/in/katelinramer/), 
+[Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), 
+e molti altri. 
+Per la lista completa fai riferimento a [questa pagina Github](https://github.com/cncf/glossary/graphs/contributors).
+
+La localizzazione del progetto in italiano è a cura di 
+[Annalisa Gennaro](https://www.linkedin.com/in/annalisagennaro/),
+[Paolo Martinoli](https://www.linkedin.com/in/paolo-e-m-martinoli/),
+[Francesco Sbaraglia](https://www.linkedin.com/in/fsbaraglia/), 
+e [Simone Stella](https://www.linkedin.com/in/simostella/).
 
```

Comment:
Matteo Bisi https://www.linkedin.com/in/matteobisi/
Sara Trappetti https://www.linkedin.com/in/sara-trappetti/

@sistella aggiungerei anche loro :)

Reply:
Fatto @annalisag-spark

Reply:
@sistella dovremmo aggiungere anche Simone Ragonesi https://www.linkedin.com/in/simone-ragonesi-275567222/ (mi ero scordata, scusa)